scripts/avn

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env $PYTHON
+#! /usr/bin/env python3
```

Comment:
I think officially dropping python support should not be part of this PR.

Reply:
Also, when this is done, we should also drop tests for python2 from travis.

Reply:
fixed